.travis.yml

```diff
@@ -5,7 +5,7 @@ scala:
 jdk:
 - oraclejdk8
 script:
-- sbt coverage test coverageReport
+- sbt coverage test coverageReport scalafmtCheck
```

Comment:
should we do it first? no need to run the tests if the formatting isn't right, wdyt?

Reply:
Sure, will move it to the front.